examples/client_side_function_calling.py

```diff
@@ -0,0 +1,297 @@
+"""This example implements a WebSocket client for interacting with Speechmatics' Flow API,
+emphasising client-side function calling using a Local Knowledge Source (`Store`)
+to manage a customer order.
+"""
+
+import asyncio
+import json
+import os
+import sys
+from typing import Any, Dict
+
+import pyaudio
+
+from speechmatics_flow.client import WebsocketClient
+from speechmatics_flow.models import (
+    ConnectionSettings,
+    Interaction,
+    AudioSettings,
+    ConversationConfig,
+    ServerMessageType,
+    ClientMessageType,
+)
+
+# Create a websocket client
+client = WebsocketClient(
+    ConnectionSettings(
+        url="wss://flow.api.speechmatics.com/v1/flow",
+        auth_token=os.getenv("SPEECHMATICS_API_KEY"),
```

Comment:
Add a readme under examples dir on setting env variables or necessary initilization steps. Should we have a check if auth_token env var is set in the examples?